From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paolo Severini <paolosev@microsoft.com>
Date: Wed, 30 Sep 2026 11:52:26 +0200
Subject: [PATCH] [upstream] [wasm interpreter] Disable unsupported mjsunit
 tests.

Bug: none
Change-Id: I2c29b725ac598607757b5a875d52402a293dda0b
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/8488465
Reviewed-by: Jakob Linke <jgruber@chromium.org>
Reviewed-by: Daniel Lehmann <dlehmann@chromium.org>
Commit-Queue: Paolo Severini <paolosev@microsoft.com>
Cr-Commit-Position: refs/heads/main@{#110186}
---
 test/mjsunit/mjsunit.status | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/test/mjsunit/mjsunit.status b/test/mjsunit/mjsunit.status
index af6517a6a8f..9b1b49ce9de 100644
--- a/test/mjsunit/mjsunit.status
+++ b/test/mjsunit/mjsunit.status
@@ -964,6 +964,8 @@
   'regress/wasm/regress-493430818': [SKIP],
   'regress/wasm/regress-493529579': [SKIP],
   'regress/wasm/regress-493521637': [SKIP],
+  'regress/wasm/regress-557846268': [SKIP],
+  'regress/wasm/regress-wasmfx-unreachable-resume': [SKIP],
   'wasm/cont-bind': [SKIP],
   'wasm/regress-496645739': [SKIP],
   'wasm/regress-497667917': [SKIP],
@@ -986,6 +988,7 @@
   'regress/wasm/regress-497330272': [SKIP],
   'regress/wasm/regress-511160771': [SKIP],
   'regress/wasm/regress-516109051': [SKIP],
+  'regress/wasm/regress-565341967': [SKIP],
   'regress/wasm/regress-crbug-1463232': [SKIP],
   'regress/wasm/regress-crbug-1466312': [SKIP],
   'regress/wasm/regress-v8-14710': [SKIP],
@@ -1012,6 +1015,7 @@
   'regress/wasm/regress-350779988': [SKIP],
   'regress/wasm/regress-366350772': [SKIP],
   'regress/wasm/regress-397043084': [SKIP],
+  'regress/wasm/regress-523591366': [SKIP],
   'wasm/denormals': [SKIP],
   'wasm/deopt/*': [SKIP],
   'wasm/enter-debug-state': [SKIP],
@@ -1074,6 +1078,7 @@
   'regress/wasm/regress-521064735': [SKIP],
   'regress/wasm/regress-524830496': [SKIP],
   'regress/wasm/regress-536581698': [SKIP],
+  'regress/wasm/regress-552026580': [SKIP],
   'regress/wasm/regress-565565747': [SKIP],
   'wasm/shared-everything/*': [SKIP],
   'wasm/subtyping-invalid': [SKIP],
@@ -1099,6 +1104,8 @@
   # --wasm-fp16
   'regress/wasm/regress-499027764': [SKIP],
   'regress/wasm/regress-504612573': [SKIP],
+  'regress/wasm/regress-565847635': [SKIP],
+  'regress/wasm/regress-simd-526303932': [SKIP],
   'wasm/fp16-load-mul': [SKIP],
   'wasm/fp16-oob-trap': [SKIP],
   'wasm/fp16-store': [SKIP],
@@ -1139,8 +1146,11 @@
   'regress/wasm/regress-457106696': [SKIP],
   'regress/wasm/regress-513063392': [SKIP],
   'regress/wasm/regress-525437482': [SKIP],
+  'regress/wasm/regress-561662099': [SKIP],
+  'regress/wasm/regress-563427048*': [SKIP],
   'wasm/custom-descriptors*': [SKIP],
   'wasm/exact-types': [SKIP],
+  'wasm/wasm-gc-js-roundtrip': [SKIP],
 
   # --wasm-compact-imports
   'wasm/compact-imports': [SKIP],
@@ -1149,6 +1159,7 @@
   'regress/wasm/regress-515525741': [SKIP],
   'regress/wasm/regress-516102112-1': [SKIP],
   'regress/wasm/regress-simd-520944504': [SKIP],
+  'wasm/wide-add': [SKIP],
   'wasm/wide-arithmetic': [SKIP],
 
   # --wasm-imported-strings-utf8
@@ -1233,11 +1244,15 @@
   'regress/wasm/regress-538378084': [SKIP],
   'regress/wasm/regress-540446645': [SKIP],
   'regress/wasm/regress-543901836': [SKIP],
+  'regress/wasm/regress-561660149': [SKIP],
   'regress/wasm/regress-crbug-1507751': [SKIP],
   'regress/wasm/regress-simd-518058891': [SKIP],
   'regress/wasm/regress-simd-525600038': [SKIP],
   'regress/wasm/regress-simd-526303929': [SKIP],
   'regress/wasm/regress-simd-528884838': [SKIP],
+  'regress/wasm/regress-simd-555760230': [SKIP],
+  'regress/wasm/regress-simd-555806042*': [SKIP],
+  'regress/wasm/regress-simd-557107349': [SKIP],
   'wasm/bit-shift-right': [SKIP],
   'wasm/bounds-check-turbofan': [SKIP],
   'wasm/enter-and-leave-debug-state': [SKIP],
@@ -1283,6 +1298,7 @@
   'regress/wasm/regress-498814570': [SKIP],
   'regress/wasm/regress-498818401': [SKIP],
   'regress/wasm/regress-500113958': [SKIP],
+  'regress/wasm/regress-555812363': [SKIP],
 
   # The interpreter call stack contains an additional top frame if we throw a
   # type-mismatch exception before executing the Wasm function.
